docs/pagination.md

```diff
@@ -0,0 +1,71 @@
+# Pagination
+
+Pagination is supported using the metadata support that is built into Ember Data. The
+metadata from Django REST Framework paginated list views is updated on every request
+to sever.
+
+
+## Retrieving the Metadata
+
+To get a page of records simply run a find request with the `page` query param:
+
+```js
+var result = this.store.find("post", {
+  page: 1
+});
+```
+
+It's possible to override the server-side page size by also including the `page_size`
```

Comment:
http://www.django-rest-framework.org/api-guide/pagination/

I just tested this and it seems to be off by default. I'll adjust the docs on this point.